src/renderer/display_list.rs

```diff
@@ -940,9 +940,20 @@ impl<'a> CursorLines<'a> {
 
 #[derive(Copy, Clone, Debug, PartialEq)]
 pub(crate) enum EndLine {
-    Eof = 0,
-    Crlf = 1,
-    Lf = 2,
+    Eof,
+    Lf,
+    Crlf,
+}
+
+impl EndLine {
+    /// The number of characters this line ending occupies in bytes.
+    pub(crate) fn len(self) -> usize {
+        match self {
+            EndLine::Eof => 0,
+            EndLine::Lf => 1,
+            EndLine::Crlf => 2,
+        }
+    }
```

Comment:
This should probably be its own PR so it can be evaluated separately

Reply:
Done, split out as #155.